backtrace-library/src/main/java/backtraceio/library/common/ApplicationHelper.java

```diff
@@ -0,0 +1,74 @@
+package backtraceio.library.common;
+
+import android.content.Context;
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
+
+import backtraceio.library.logger.BacktraceLogger;
+
+public class ApplicationHelper {
```

Comment:
WDYT about naming that e.g. `AppInfoCache`/`ApplicationMetadata`/`CachedAppInfo`

Reply:
Makes sense

Reply:
Renamed it to ApplicationMetadataCache